apps/wallet-mobile/src/legacy/config.ts

```diff
@@ -46,7 +47,7 @@ export const LINKING_CONFIG = {
               initialRouteName: 'history-list',
               screens: {
                 'exchange-result': {
-                  path: RAMP_ON_OFF_PATH,
+                  path: configYoroiExchangeOrderShowCreateResult.path,
```

Comment:
can you confirm this @stackchain ?

Reply:
this can be dropped there is a hook that takes over and redirect now, useLinksShow....